src/linearmixedmodel.jl

```diff
@@ -159,15 +159,21 @@ fit(
     REML = REML,
 )
 
-StatsBase.coef(m::MixedModel) = fixef(m, false)
+function StatsBase.coef(m::LinearMixedModel{T}) where {T}
+    piv = fetrm(m).piv
+    invpermute!(fixef!(similar(piv, T), m), piv)
+end
 
-βs(m::LinearMixedModel) = NamedTuple{(Symbol.(fixefnames(m))...,)}((fixef(m)...,))
+βs(m::LinearMixedModel) = NamedTuple{(Symbol.(coefnames(m))...,)}(coef(m))
```

Comment:
`GeneralizedLinearMixedModel` has a property `beta`: not the quite the same as the accessor here, but should standardize? And/or do a `MixedModel` method instead -- `coefnames` and `coef` are defined for both GLMM and LMM.

Reply:
I think it is best to keep separate methods for `LinearMixedModel` and `GeneralizedLinearMixedModel`.  When beta is part of the "baba yaga" optimization you can't guarantee that the calculation using only the LMM field will produce the correct answer.  This is related to the distinction between `fast=true` and `fast=false`.